README.md

```diff
@@ -30,19 +31,49 @@ type: docs
 
 ## Adding release-specific docs
 
-Steps to add a release-specific version of the docs site:
+### Create a release branch
 
-1. Create a release branch from `main` in your fork of this repo. Name this new branch after the released major and minor version, like [release-v0.8](https://github.com/openservicemesh/osm-docs/tree/release-v0.8).
-1. Update the site's config.toml file in your branch to reflect the release version - add [these params](https://github.com/openservicemesh/osm-docs/blob/release-v0.8/config.toml#L99L101) to enable the banner at the top of the site that will tell visitors which version they are looking at.
-1. List the new branch in the [config.toml](https://github.com/openservicemesh/osm-docs/blob/main/config.toml#L84-L89) for all currently-displayed versions.
-1. Push your newly-created `release-X.Y` branch upstream to this repo on GitHub in order for the new branch to be usable by Netlify
-1. Netlify will auto-deploy the branch to a url like [https://release-v0-8--osm-docs.netlify.app/](https://release-v0-8--osm-docs.netlify.app/). Use this to preview and test that this branch builds correctly.
-1. Open an issue in this repo asking for a new DNS record be added to the site (via Netlify), to assign a subdomain to the deployed branch.
-1. When published, the newly-added branch will function like [https://release-v0-8.docs.openservicemesh.io/](https://release-v0-8.docs.openservicemesh.io/)
+Look for a branch in the upstream repo named `release-vX.Y`, where `X` and `Y` correspond to the major and minor version of the new release. For example, [release-v0.8](https://github.com/openservicemesh/osm-docs/tree/release-v0.8). If the branch already exists, move to the next step.
+
+Identify the base commit in the main branch for the release and cut a release branch off main.
+
+> Note: Care must be taken to ensure the release branch is created from a commit meant for the release. If unsure about the commit to use to create the release branch, please open an issue in the osm repo and a maintainer will assist you with this.
+
+```console
+$ git checkout -b release-<version> <commit-id> # ex: git checkout -b release-v0.4 0d05587
+```
+
+Push the release branch to the upstream repo (NOT forked), identified here by the upstream remote.
+
+```console
+$ git push upstream release-<version> # ex: git push upstream release-v0.4
+```
+
+Netlify will auto-deploy the branch to a url like [https://release-v0-8--osm-docs.netlify.app/](https://release-v0-8--osm-docs.netlify.app/). This can be used to preview and test that the branch builds correctly.
+
+### Update the release branch
+
+1. Create a new branch off of the release branch to maintain updates specific to the new version. Let's call it the patch branch. The patch branch should not be created in the upstream repo.
```

Comment:
Do you want to also note that it's recommended to avoid using the same branch name as the release tag because that could result in problems when pushing the tag?

Reply:
We currently don't use tags in the osm-docs repo. This note is relevant for the osm repo.